core/oslib/audiobackend_alsa.cpp

```diff
@@ -188,7 +188,7 @@ std::vector<std::string> alsa_get_devicelist()
 		return result;
 
 	// special value to automatically detect on initialization
-	result.push_back("auto");
+	result.emplace_back("auto");
```

Comment:
Unless there is a good reason, I prefer to keep "good old c++" rather than using more recent c++ features just for the sake of it.